nrf_802154_sl/Kconfig

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if NRF_802154_RADIO_DRIVER
+
+choice NRF_802154_SL_CHOICE
```

Comment:
Is the intention that there will different flavors of the NRF_802154_SL?

Reply:
Yes. There will be version in upsteram hal_nordic, working with pure zephyr called "open source" or "community". It will have reduced or simplified set of features. Full-featured version will be available as binary in NCS through nrfxlib. Upstream does not allow binaries, we need IP protection, but still want t have upstream version working to some extent.